packages/dd-trace/src/debugger/devtools_client/index.js

```diff
@@ -45,6 +45,7 @@ session.on('Debugger.paused', async ({ params }) => {
   let sampled = false
   let numberOfProbesWithSnapshots = 0
   const probes = []
```

Comment:
suggest to have a single array of objects to avoid out of sync errors between probes and errors. also enable to avoid two push calls.

`const probesEval = {probe:, eval: }[]`

Reply:
yeah. I think it is ok and now that we also have getDDTagsExpression in there my suggestion will not work.

it is added complexity but I think it is handleable.